source/dem/particle_point_line_contact_force.cc

```diff
@@ -40,8 +40,9 @@ ParticlePointLineForce<dim>::calculate_particle_point_line_contact_force(
       // are used to consider non-linear relation of the contact force to
       // the normal overlap
       double model_parameter_betha =
-        log(physical_properties.poisson_ratio_wall) /
-        sqrt(pow(log(physical_properties.poisson_ratio_wall), 2.0) + 9.8696);
+        log(physical_properties.restitution_coefficient_wall) /
+        sqrt(pow(log(physical_properties.restitution_coefficient_wall), 2.0) +
```

Comment:
```suggestion
        sqrt(pow(log(physical_properties.restitution_coefficient_wall), 2) +
```

When you have a power 2, use 2. The compiler can then optimize it more.